modules/ossm-installing-cert-manager.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-installing-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-installing-cert-manager_{context}"]
+= Installing cert-manager
+//TP1 content influx. Title, etc may change.
+//Content is very similar to 2.x content
+//all kinds of formatting things to fix. want to see if a build will generate to have a look, and see how it fits structurally with the IA.
+
+You can integrate `cert-manager` with {SMProduct} by deploying `istio-csr` and then creating an `Istio` resource that uses `istio-csr` to process workload and `controlplane` certificate signing requests. This example creates a self-signed `Issuer`, but any other `Issuer` can be used instead.
```

Comment:
I've mentioned this at least once earlier, but both cert-manager and istio-csr are referred to inconsistently. Too avoid confusion, it's best to pick one phrase for when you are referring to the general name of the product/project and another when you are referring to the specific binary/service/object/application. And a reminder that if you use backtics, you need to follow the phrase in backtics with a noun.

There are other comments that address some of this in more detail or show examples.